pytorch_lightning/utilities/__init__.py

```diff
@@ -23,36 +23,16 @@
 
 from pytorch_lightning.utilities.apply_func import move_data_to_device
 from pytorch_lightning.utilities.distributed import AllGatherGrad, rank_zero_info, rank_zero_only, rank_zero_warn
+from pytorch_lightning.utilities.package_utils import _module_available
```

Comment:
well, this was moved already to #5256 along with all COnstants so we can import them anywhere without cyclic imports...